creating_images/sti.adoc

```diff
@@ -17,10 +17,11 @@ output.
 
 The main advantage of using STI for building reproducible Docker images is the
 ease of use for developers. As a builder image author, you must be aware of the
-two basic requirements for the best possible STI performance: the required image
-contents and STI scripts.
+two basic requirements for the best possible STI performance: link:#build-process[the build process]
```

Comment:
maybe that's in your other PR. i'm going to ignore the rest of the STIs in this PR.